Library/Formula/silk.rb

```diff
@@ -17,8 +15,12 @@ def install
                           "--enable-ipv6",
                           "--enable-data-rootdir=#{var}/silk"
     system "make"
-    system "make install"
+    system "make", "install"
 
     (var+"silk").mkpath
   end
+
+  test do
+    system "#{bin}/silk_config", "--version"
```

Comment:
Could maybe use this to read/output some dummy data: https://tools.netsa.cert.org/silk/rwcat.html